src/commandLine.js

```diff
@@ -119,8 +119,13 @@ var CommandLineManager = {
     index = argv.indexOf('--proxy-host');
     if (index !== -1) {
       if (CommandLineManager.validArgValue(argv[index + 1])) {
+        var hostRegex = /(^\w+:|^)\/\//;
```

Comment:
```suggestion
        var protocolRegex = /(^\w+:|^)\/\//;
```
and this can be moved to constants as well as it can be re-used at other places if required.

Reply:
Done.